js/analyzer.js

```diff
@@ -401,12 +401,7 @@ Analyzer.prototype.urlKeyword = function() {
  * @returns {{test: string, result: number}[]}
  */
 Analyzer.prototype.urlLength = function() {
-	var result = [ { test: "urlLength", result: { urlTooLong: isUrlTooLong(
-		this.paper.getUrl(),
-		this.paper.getKeyword(),
-		this.config.maxSlugLength,
-		this.config.maxUrlLength
-	) } } ];
+	var result = [ { test: "urlLength", result: { urlTooLong: isUrlTooLong( this.paper ) } } ];
```

Comment:
This will be indeed deprecated, the analyzer will be removed if the assessments are finished.